Makefile

```diff
@@ -50,6 +50,9 @@ itest:
 
 qcows:
 	cp dist/artifacts/rancheros.iso scripts/images/openstack/
+	cd scripts/images/openstack && \
```

Comment:
I want to move these builds to [os-packer](https://github.com/rancher/os-packer).
Can you please try to make a PR for the cloudstack image?

Reply:
Ah great, didn't know that repo. I'll remove the code here and add another PR over there.